### PR TITLE
[iOS] Added icon to context menu actions instead of text [BROKEN]

### PR DESCRIPTION
**Pull request by Sebastian1989101**
### Description of Change ###

Changed context action menu of ViewCells to Icons like it is in native apps and on Android instead of Text. This only happens if the ListViewRenderer is overwritten and the virtual protected property `ActionCellIconInsteadOfText` is set to `true`. This way this is not a breaking change as you mentioned here: https://github.com/xamarin/Xamarin.Forms/pull/1487

### Bugs Fixed ###

- None

### API Changes ###

 - None

### Behavioral Changes ###

Instead of text the icon will appear in context actions like it is on Android.

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs [hunk 1]
```diff
@@ -44,6 +44,8 @@ protected UITableViewRowAnimation ReloadSectionsAnimation
 			set { _dataSource.ReloadSectionsAnimation = value; }
 		}
 
+		protected virtual bool ActionCellIconInsteadOfText { get; }
```
Comment (rmarinho) [inline Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs]: Not sure about the naming  `ContextActionShouldUseIcon` ?!

Comment (Sebastian1989101) [reply Xamarin.Forms.Platform.iOS/Renderers/ListViewRenderer.cs]: You can name it anything you want. It just shows where changes are required for a minimal change and without changing the behavior on any existing app.

**Comment by rmarinho** [comment]
@Sebastian1989101 sorry can you rebase so we can fix the certs to run on OSX. Thanks

**Comment by Sebastian1989101** [comment]
@rmarinho hmm did I something wrong with the rebase? Because it now shows all the changes from the others within here.

**Comment by rmarinho** [comment]
yeah @Sebastian1989101 something went wrong there..

**Comment by Sebastian1989101** [comment]
@rmarinho Is there a way to delete this PR and make it new from scratch? Because now it is +3,391 −775 for the commits and I dunno why.

**Comment by rmarinho** [comment]
@Sebastian1989101 yeah, so you makse sure yuu have master branch updated with latest stuff, then you create a new branch from master and call it icon-feature for exemple, then you cherry pick the commits you want from the old branch , after that you force push the origin of you inital branch that is point the pr, for example master

**Comment by Sebastian1989101** [comment]
@rmarinho I have redone the PR, this time with a own branch, here: #2629 so this one here can be removed/closed due to a broken rebase.

**Pull request closed without merging**